src/librustdoc/html/static/settings.css

```
@@ -59,3 +67,9 @@ input:checked + .slider:before {
 	-ms-transform: translateX(19px);
 	transform: translateX(19px);
 }
+
+.setting-line > .sub-setting {
```

To match the change to `sub-settings` you'd need:
```suggestion
.setting-line > .sub-settings {
```